Author: Adam Gandelman <adamg@ubuntu.com>
Date: Tue Jul  2 17:42:44 PDT 2013
Subject: Remove upper limit on requests dependency.

Upstream capped the requests version due to various reasons related
to a centos failing python 2.6 gating job.  We package and ship
python-requests 1.2.3 in 13.10.

For context of upstream change, see:

https://bugs.launchpad.net/nova/+bug/1182271

Index: nova-git/requirements.txt
===================================================================
--- nova-git.orig/requirements.txt	2013-07-02 17:37:32.291186196 -0700
+++ nova-git/requirements.txt	2013-07-02 17:38:56.495190211 -0700
@@ -20,7 +20,7 @@
 pyasn1
 Babel>=0.9.6
 iso8601>=0.1.4
-requests>=1.1,<1.2.1 # order-dependent python-cinderclient req cap, bug 1182271
+requests>=1.1 # order-dependent python-cinderclient req cap, bug 1182271
 python-cinderclient>=1.0.1
 python-quantumclient>=2.2.0,<3.0.0
 python-glanceclient>=0.5.0,<2
